ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -303,32 +303,34 @@ class IdpLoginScreen extends React.Component<Props, State> {
           loggedOutWithIdpAuth.idp.name
         }`}
       >
-        {!hasError && (
+        {
```

Comment:
Is there a reason I don't see for these braces here? The JSX should work fine without them, what do you think? 🙂

Reply:
🦅 
Thank, remove here 0d7adb880bf9ea03cb5b973d7bee7fff83d2b52e